controllers/rest/global_initializers.go

```diff
@@ -0,0 +1,19 @@
+package rest
+
+import (
+	"github.com/getkin/kin-openapi/openapi3"
+	weosContext "github.com/wepala/weos/context"
+	"golang.org/x/net/context"
+)
+
+//Security adds authorization middleware to the initialize context
+func Security(ctxt context.Context, api *RESTAPI, swagger *openapi3.Swagger) (context.Context, error) {
+	middlewares := GetOperationMiddlewares(ctxt)
+	if swagger.Security != nil && len(swagger.Security) != 0 {
+		if middleware, _ := api.GetMiddleware("AuthorizationMiddleware"); middleware != nil {
```

Comment:
This should only add the middleware if the type is "openIdConnect" (maybe the middleware should be renamed OpenIDMiddleware since technically it's specific to that flow.